### ISSUE TITLE
Skip download ads without pressing Skip Ad button

### ISSUE THREAD

**Issue by BluestoneDE** [MartinBraquet/youtube-adblock#12]
## Summary
This is one remaining problem from #8.
The addon won't skip download links if they exist at the end of an ad.

## Reproduction
1. In the popup set **If Skip Available** to **Boost Ad**
2. Go on [YouTube]
3. Click on any video with ads
4. Keep refreshing until an ad shows up with a download

## Expected Behavior
The ad gets skipped completely without clicking "Skip Ad".

## Actual Behavior
The ad isn't skipped completely.

## Screenshots
![screenshot]

## Environment
 - OS: Windows 10
 - Firefox Version: 127.0.2
 - Addon Version: 1.70

[YouTube]: https://www.youtube.com
[screenshot]: https://github.com/MartinBraquet/youtube-adblock/assets/65172029/02b341e2-35c4-45e9-8425-3b964c97dfec

**Comment by MartinBraquet** [comment]
Thanks for keeping an issue open about this. Maybe clicking on the button at that moment would work and not be detected.

**Comment by BluestoneDE** [comment]
Nope. I already tried that. The button cannot be clicked via a script. It's very likely that YouTube checks the `isTrusted` property. Therefor it makes more sense to speed up the automatic piechart timer instead. But I got no idea where to begin with this.

**Comment by MartinBraquet** [comment]
Got it, I opened a wiki and project to jot down ideas and interesting info, like the current status of the youtube detection algorithm (what they detect, etc). Both are aimed to be fully open access so everyone can contribute without the burden of reviewing the info. Feel free to use either of them if that helps.

**Comment by MartinBraquet** [comment]
I won't be looking much into this issue if we don't get many complaints from users, but feel free to look into it in the future if you have any interest.

**Comment by Archaeologistflash** [comment]
I think you'll get those complaints. Whether I click the skip button, or wait for the timer to go around, about 25% of the time the video loses the image and resumes with audio only. And refreshing the page sends me back to the start of the video. It's unwatchable to have to keep refreshing and then finding where I'd got to.
